MVC-App/Controllers/tblRelationsController.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Entity;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using MVC_App.Domain.Models;
+
+namespace MVC_App.Controllers
+{
+    public class tblRelationsController : Controller
+    {
+        private testEntities db = new testEntities();
+
+        private IEnumerable<RelationModel> relationModels;
+
+        public tblRelationsController()
+        {
+            var relations = from tblRelation in db.tblRelation
```

Comment:
simplify
relationModels =  from tblRelation in db.tblRelation ......... .ToList();